From ce24f4787ea5647be849590f7eff6f6c1951b504 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sun, 12 Sep 2021 21:04:40 +0200
Subject: [PATCH 1/2] Add back conda and conda_env entry point

Partially reverts 0ccc029997e0dc0a28420a89e0cb39c08ff0b738.
'conda init' is designed to fail and emit a warning. Go back
to the normal init.
---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 84ecb55cca..bb3db39c4d 100644
--- a/setup.py
+++ b/setup.py
@@ -86,7 +86,8 @@ setup(
     },
     entry_points={
         'console_scripts': [
-            'conda=conda.cli.main_pip:main',
+            'conda=conda.cli.main:main',
+            'conda-env = conda_env.cli.main:main',
         ],
     },
     install_requires=install_requires,
-- 
2.32.0
